src/4.3_to_5.0/accumulateMigratedKpiTitles.ts

```diff
@@ -43,49 +43,55 @@ function _getLegacyKpiTitles(
     } = {};
     const memberUniqueNames = tupleKey.split(/,(?![^\[]*\])/);
     for (const memberUniqueName of memberUniqueNames) {
-      const compoundIdentifier =
-        parse<MdxUnknownCompoundIdentifier>(memberUniqueName);
-      const specificCompoundIdentifier = getSpecificCompoundIdentifier(
-        compoundIdentifier,
-        { cube },
-      );
-      if (specificCompoundIdentifier.type === "measure") {
-        tuple[`[Measures].[Measures]`] = [
-          specificCompoundIdentifier.measureName,
-        ];
-      } else if (specificCompoundIdentifier.type === "member") {
-        const hierarchyUniqueName = quote(
-          specificCompoundIdentifier.dimensionName,
-          specificCompoundIdentifier.hierarchyName,
+      if (memberUniqueName) {
+        const compoundIdentifier =
+          parse<MdxUnknownCompoundIdentifier>(memberUniqueName);
+        const specificCompoundIdentifier = getSpecificCompoundIdentifier(
+          compoundIdentifier,
+          { cube },
         );
-        tuple[hierarchyUniqueName] = specificCompoundIdentifier.path;
+        if (specificCompoundIdentifier.type === "measure") {
+          tuple[`[Measures].[Measures]`] = [
+            specificCompoundIdentifier.measureName,
+          ];
+        } else if (specificCompoundIdentifier.type === "member") {
+          const hierarchyUniqueName = quote(
+            specificCompoundIdentifier.dimensionName,
+            specificCompoundIdentifier.hierarchyName,
+          );
+          tuple[hierarchyUniqueName] = specificCompoundIdentifier.path;
+        }
```

Comment:
Looks like a big change but all I did was wrap this logic block in an `if` statement.